pants.toml

```diff
@@ -24,6 +24,7 @@ backend_packages.add = [
   "pants.backend.experimental.java.lint.google_java_format",
   "pants.backend.experimental.java.debug_goals",
   "pants.backend.experimental.python",
+  "pants.backend.experimental.python.packaging.pyoxidizer",
```

Comment:
I feel like a `packaging` (or some other namespace) is required, to line up with lint.

For example, I am considering making a Nuitka plugin, which would operate similarly to PyOxidizer - so they should have -some- shared top-level.

Reply:
Seems reasonable to me 👍🏽